data.go

```
@@ -479,10 +506,18 @@ func getApidInstanceInfo() (info apidInstanceInfo, err error) {
 		newInstanceID = true
 		info.InstanceID = GenerateUUID()
 
-		db.Exec("REPLACE INTO APID (instance_id, apid_cluster_id, last_snapshot_info) VALUES (?,?,?)",
+		_, err = tx.Exec("REPLACE INTO APID (instance_id, apid_cluster_id, last_snapshot_info) VALUES (?,?,?)",
```

This err is not checked.

---

Err value is checked. See below...